deploy/playbooks/roles/common/templates/prod.py.j2

```diff
@@ -118,6 +118,19 @@ purge_rotation = {
             'keep_minimum': 10
         }
     },
+    'ceph-installer' {
+        'master': {
+            'keep_minimum': 10
+        },
+    },
+    'ceph-ansible' {
+        'master': {
+            'keep_minimum': 5
+        },
+        'v2.1.0': {
```

Comment:
It's also a branch https://github.com/ceph/ceph-ansible/tree/v2.1.0

This was the branch we planned to use for downstream support.